### PR TITLE
6 completely rework readme

### LINKED ISSUE #6: Completely rework readme

### PR DESCRIPTION
**Pull request by Danilado**
#6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,29 @@
-# Копия итогового проекта с курса "промышленное программирование" [школы программистов](https://informatics.ru)
----
+# The final project for the "Industrial Programming" course of [MSHP](https://informatics.ru)
```
Comment (quswadress) [inline README.md]: I would make a separate Russian translation of the readme and move this disclaimer there. Nobody outside Moscow and its districts needs this disclaimer. Mention that this is a training project is already written below.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
 
-# Baba is u limited Жaba edition with new level, textures, multiplayer, open-source, and best refactored code.
-## Концепция
+# The original game is called [Baba is You](https://store.steampowered.com/app/736260/Baba_Is_You/) 
+We don't take any credit on creating the game idea and /or mechanics. This is a project created for educational purposes with consent of original game's creator (Hempuli) through an email
```
Comment (quswadress) [inline README.md]:
```suggestion
We don't take any credit for creating the game's idea, mechanics, sprites, and/or music. This is a project created for educational purposes with the consent of the original game's creator ([Hempuli](https://www.hempuli.com/)) via email.
```

**Review by quswadress — DISMISSED** [review]

**Review by Danilado — COMMENTED** [review]
done
https://github.com/Danilado/jaba-is-you/pull/11/commits/5b6f2ae5e1d668011abbc01f305547527f821ec3

**Review by quswadress — APPROVED** [review]
Looks good, _at least adequate_. :-)

**Pull request merged**